novuhq/website
# chore: migrating a blog from Strapi to Wordpress

**Describe what changes this pull request brings**
This pull request adds migrating a blog from `Strapi` to `Wordpress`.

**Describe any additional changes**

- updating `RSS` file generation

**Steps to test**
1. Open [the page](https://websitemain13551-wordpressblog.gtsb.io/blog/)
2. Confirm that everything looks and works as expected
